cypress/integration/WhereStartLookingPage.feature

```diff
@@ -12,3 +12,13 @@ Feature: Where should we start looking Page
   Scenario: Has a back link to navigate to the how did you lose contact page
     When I click the "back" link
     Then I am on the "how did you lose contact with your family" page
+
+  Scenario: Selecting One Country along with an active country and clicking check eligibility navigates Relative In Hiding page
+    When I select "one country in particular" radio button
+    When I click the "check eligibility" button
+    Then I am on the "relatives in hiding" page
+
+  Scenario: Selecting Two or Three Countries clicking check eligibility navigates Relative In Hiding page
+    When I select "two or three countries" radio button
+    When I click the "check eligibility" button
+    Then I am on the "relatives in hiding" page
```

Comment:
The prototype does this...

![image](https://user-images.githubusercontent.com/8738245/89780624-ba006c80-db09-11ea-8022-92ee49ddb2fc.png)

I'm wondering why we need the one country versus 2 or 3 if the result is the same. Why don't we just let them select up to three?